src/components/organisms/ChatSidebar/components/WorldChat/WorldChat.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
```

Comment:
I see that you have `VenueChat` (which has all of the current `ChatList`/`ChatMessage`/etc underneath it, and you also have `WorldChat` here. Are you intending to duplicate those `ChatList`/`ChatMessage`/etc components underneath here as well? If so, that's definitely not something we want to do. Venue/World/etc chats should all basically be a tiny bit of code that connects them to their data source, but otherwise uses exact same generic underlying 'dumb components' ('dumb' being that they don't connect to the stores/etc by themselves, and are entirely driven by the props passed into them)